CMakeLists.txt

```
@@ -27,8 +27,13 @@ mark_as_superbuild(Slicer_DIR)
 find_package(Git REQUIRED)
 mark_as_superbuild(GIT_EXECUTABLE)
 
-find_package(ExtraMarkups REQUIRED)
-mark_as_superbuild(ExtraMarkups_DIR)
+option(SlicerVMTK_enable_ExtraMarkups_and_StenosisMeasurement3D
```

This should be sufficient (if `ExtraMarkups` is disabled we could then disable all modules that depend on it).
```suggestion
option(SlicerVMTK_USE_ExtraMarkups
```